Improve PR template wording (#1122)
fastlane/docs: .github/PULL_REQUEST_TEMPLATE.md

<!--

> Thanks for contributing to _fastlane_! Before you submit your pull request, note that files in the docs folder covering Actions (actions.md and actions/*) and Plugins (available-plugins.md) are auto-generated.

Reviewer: This brought up to my attention that we've already moved all of them to `docs/Generated` directory. So:

```suggestion
Thanks for contributing to _fastlane_! Before you submit your pull request, note that files in the `/docs/generated` folder are auto-generated.
```

> If this is the case, please modify the documentation at their sources (https://github.com/fastlane/fastlane or plugin repository) and it will be updated here regularly.

Author: Or maybe something like:

```suggestion
If this is the case, please modify the documentation at its source (at https://github.com/fastlane/fastlane or plugin repository) and it will propagate here on regular basis.
```

Reviewer: I like this!
